File: QTD.SPA/src/app/_DtoModels/Segment_ObjectiveLink/SegmentObjectiveOrderVM.ts
export class SegmentObjectiveOrderVM {
    segmentObjectiveLinkId: string; 
    order: number; 
    type: string; 
    objectiveId : string | null; 
  
    constructor(
      _segmentObjectiveLinkId: string,
      _order: number,
      _type: string,
      _objectiveId: string | null
    ) {
      this.segmentObjectiveLinkId = _segmentObjectiveLinkId;
      this.order = _order;
      this.type = _type;
      this.objectiveId = _objectiveId;
    }
  }
  

File: QTD2.Infrastructure/Model/SegmentObjective_Link/SegmentObjectiveOrderVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.SegmentObjective_Link
{
    public class SegmentObjectiveOrderVM
    {
        public int? SegmentObjectiveLinkId { get; set; } 
        public int ObjectiveId { get; set; }
        public string Type { get; set; } 
        public int Order { get; set; } 

        public SegmentObjectiveOrderVM() { }
    }
}
